rfcs/text/0002_encrypted_attributes.md

```diff
@@ -0,0 +1,228 @@
+- Start Date: 2019-03-22
+- RFC PR: https://github.com/elastic/kibana/pull/34526
```

Comment:
I believe this is suppose to link to _this_ PR, not the implementation. The idea being that in the future you can easily find this RFC PR which might contain important discussion/context.

Reply:
Ah, silly me. Will change, thanks!